docs/content/en/docs/clustermgmt/cluster-backup-restore/backup-cluster.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Backup cluster"
+linkTitle: "Backup cluster"
+weight: 20
+aliases:
+    /docs/tasks/cluster/cluster-backup-restore/backup-cluster/
+description: >
+  How to backup your EKS Anywhere cluster
+---
+
+We strongly advise performing regular cluster backups of all the EKS Anywhere clusters under your management. This ensures that you always have an up-to-date cluster state available for restoration in case the cluster experiences issues or becomes unrecoverable. This document outlines the steps for creating the two essential types of backups required for the [EKS Anywhere cluster restore process]({{< relref "./restore-cluster" >}}).
+
+## Etcd backup
+
+For optimal cluster maintenance, it is crucial to perform regular etcd backups on all your EKS Anywhere management and workload clusters. In the event of a cluster upgrade failure, make sure to **always** execute the etcd backup beforehand. To create an etcd backup for your cluster, follow the guidelines provided in the [External etcd backup and restore]({{< relref "../etcd-backup-restore/etcdbackup" >}}) section.
+
+
+## Cluster API backup
+
+Since cluster failures primarily occur following unsuccessful cluster upgrades, EKS Anywhere takes the proactive step of automatically creating backups for the Cluster API objects that capture the states of both the management cluster and its workload clusters. These backups are stored within the management cluster folder, where the upgrade command is initiated from the Admin machine, and are generated before each upgrade process.
+
+Although the likelihood of a cluster failure occurring without any associated cluster upgrade operation is relatively low, it is still recommended to manually back up these Cluster API objects on a routine basis. For example, to create a Cluster API backup of a cluster:
+
+
+```bash
+MGMT_CLUSTER="mgmt"
+MGMT_CLUSTER_KUBECONFIG=${MGMT_CLUSTER}/${MGMT_CLUSTER}-eks-a-cluster.kubeconfig
+BACKUP_DIRECTORY=backup-mgmt
+
+# Substitute the container version with whatever EKS Anywhere CLI version you are using
+CONTAINER=public.ecr.aws/eks-anywhere/cli-tools:v0.16.2-eks-a-41
```

Comment:
Can we provide a cmd here to set the `CONTAINER`? Most users won't have an easy way of deducing this image.
@abhay-krishna can you provide a fancy cmd for this?